### PR TITLE
Try another fixes

### PR DESCRIPTION
**Pull request by Dantemss**
- Made try another's transaction serializable and added locks to prevent double clicks from creating multiple problems
- Add a recovery_group group_type for try another/refresh my memory
- Convert existing default_group steps to recovery_group
- Specs

### CODE REVIEW

**Comment by Dantemss** [comment]
Closing and reopening for Travis

**Comment by jpslav** [comment]
@Dantemss please rebase

**Comment by Dantemss** [comment]
@jpslav done

File: app/subsystems/tasks/recover_task_step.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Tasks::RecoverTaskStep
 
-  lev_routine
+  lev_routine transaction: :serializable
```
Comment (jpslav) [inline app/subsystems/tasks/recover_task_step.rb]: wow -- the big guns! ;-)

**Pull request merged**